spec/serializers/activitypub/actor_serializer_spec.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe ActivityPub::ActorSerializer do
```

Comment:
Was trying to fix this message in https://github.com/mastodon/mastodon/commit/c1cb3d58dcaf7961f6b3ab32be9dcbe795688dca, but wasn't sure if it was just better to do the rename of the folder instead

Reply:
I don't think the folder should be renamed. I think it should be consistent with the autoloading and the inflections defined in `config/initializers/inflections.rb`